src/Select/elements.js

```diff
@@ -95,7 +98,15 @@ export const MenuItem = styled.li`
   }
 
   cursor: pointer;
-  :hover {
-    background: ${({ theme: { palette } }) => palette.veryLightBlue} 0%;
-  }
+
+  ${({ isSelected }) =>
+    isSelected
+      ? css`
+          background: ${({ theme: { palette } }) => palette.veryLightBlue};
+        `
+      : css`
+          :hover {
+            background: hsla(220, 27%, 94%, 0.5);
```

Comment:
Using a hardcoded value here (which is in fact `palette.veryLightBlue`) because I needed to use `hsla`. I'm taking notes about that so maybe we'll improve our design system and the way the palette colours works

Reply:
You can use https://polished.js.org/docs/#transparentize